src/scripts/main.js

```diff
@@ -6,3 +6,17 @@ const people = require('./lib/people');
 console.log(people); // you can remove it
 
 // write your code here
+const dashboard = document.querySelector('.dashboard tr');
+
+people.forEach(person => {
+  dashboard.insertAdjacentHTML('afterend', `
+  <tr>
+    <td>${person.name}</td>
+    <td>${person.sex}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${person.died - person.born}</td>
+    <td>${Math.ceil(person.died / 100)}</td>
+  </tr>
+    `);
```

Comment:
```suggestion
  dashboard.insertAdjacentHTML('afterend', `
    <tr>
      <td>${person.name}</td>
      <td>${person.sex}</td>
      <td>${person.born}</td>
      <td>${person.died}</td>
      <td>${person.died - person.born}</td>
      <td>${Math.ceil(person.died / 100)}</td>
    </tr>
  `);
```

Reply:
https://aliliolek.github.io/js_task_generate_table_DOM/

Reply:
Not fixed